diff --git a/util/pigpiod.service b/util/pigpiod.service
index f040d38..a8ec525 100644
--- a/util/pigpiod.service
+++ b/util/pigpiod.service
@@ -1,10 +1,10 @@
 [Unit]
-Description=Pigpio daemon
+Description=Daemon required to control GPIO pins via pigpio
 
 [Service]
-Type=forking
-PIDFile=pigpio.pid
-ExecStart=/usr/bin/pigpiod
+Type=simple
+ExecStart=/usr/bin/pigpiod -l -g
+SuccessExitStatus=0 255
 
 [Install]
 WantedBy=multi-user.target
